gui/src/components/onboarding/pages/body-proportions/scaled-steps/ManualHeightStep.tsx

```diff
@@ -123,6 +123,16 @@ export function ManualHeightStep({
                 doubleStep={0.1}
               />
             </form>
+            <div className="flex flex-col self-center items-center justify-center">
+              <Typography>
+                {l10n.getString(
+                  'onboarding-scaled_proportions-manual_height-estimated_height'
+                )}
+              </Typography>
+              <Typography>
+                {mFormat.format(control._formValues.height * 0.936)}
```

Comment:
you should use the [watch](https://react-hook-form.com/docs/useform/watch) function from the `useForm`

Reply:
ty, is this better?